### ISSUE TITLE
v2.0.5b - ESPs not loading

### ISSUE THREAD

**Issue by WhiteZeroX** [TanninOne/modorganizer#697]
I haven't tried MO or loading FO4 at all since before the 1.5 update dropped. But I did a clean install of v2.0.5b (just copied over my mods and profiles folders from the old MO installs) and gave it a try. Upon trying to load one of my game, it said I was missing the 4 ESPs I have loaded and checked in MO.

However, non-ESP based mods, like Full Dialogue Interface, seem to still be loading and work. 

[logs.zip](https://github.com/TanninOne/modorganizer/files/258390/logs.zip)

**Comment by KeltecRFB** [comment]
Same problem as #689

**Comment by WhiteZeroX** [comment]
I'd just like to add that when I launch LOOT through MO, all of my ESP do show up there, even though they aren't loading inside of FO4.

**Comment by WhiteZeroX** [comment]
Ok, I must have had something broken in a previous Profile. I deleted and created my MO profiles from scratch, and they seem to work fine in B2 now.

**Comment by WhiteZeroX** [comment]
Correction, I can only get my ESPs to work if I run LOOT and then run the game. If I launch MO and directly run the game, my ESPs won't load.

**Comment by WhiteZeroX** [comment]
@TanninOne I did some more testing, it seems like this issue is caused by MO adding Fallout3.esm to my plugins.txt list. If I remove that line from plugins and launch the game, my mod ESP load in just fine. But of course every time I launch MO, it adds Fallout4.esm to my plugins again.

**Comment by fingerboxes** [comment]
@WhiteZeroX I am having the same issue, no ESPs other than Official DLC seem to load. I cannot duplicate your workaround, though.

**Comment by WhiteZeroX** [comment]
Update: Still having the same issue in 2.0.7

**Comment by KeltecRFB** [comment]
Not entirely, the profile INIs appear to be loading but it appears the save games are not.  I will post log 2.0.4 log files tomorrow.  Here are 2.0.7's

[usvfs-2016-06-19_21-10-01.zip](https://github.com/TanninOne/modorganizer/files/322711/usvfs-2016-06-19_21-10-01.zip)

**Comment by WhiteZeroX** [comment]
@TanninOne  Another update, I've discovered that MO adds Fallout4.esm to my plugins.txt list only when launching the game via F4SE through MO. If I launch Fallout 4 without F4SE, MO dosen't add Fallout4.esm and my loads load fine.

**Comment by fingerboxes** [comment]
Check your symlinks. This went away for me with an OS reinstall when I
stopped symlinking C:/Games/ to .../steamapps/common/... and did it the
other way around instead.

**Comment by WhiteZeroX** [comment]
@fingerboxes I don't do any manual symlinks

**Comment by Gernash** [comment]
are you installing Beth mods or using NMM at the same time?

**Comment by WhiteZeroX** [comment]
@gernash neither, I download directly off of Nexus  Mods into MO and install from there.

**Comment by Gernash** [comment]
have you changed the default mod path in any way in MO2 Settings?

**Comment by WhiteZeroX** [comment]
@gernash Can't remember exactly, possibly. I store the active mods and profiles in the Fallout 4 game dir, as I've tried with both the install and portable modes  of MO.

**Comment by Gernash** [comment]
Wont work known issue....(

how to bodgy it up
http://wiki.step-project.com/User:Gernash/MODWiki#Converting_to_My_wiki_is_easy.....

Recommend redoing it all and follow the cheat sheet at least

**Comment by Gernash** [comment]
NOTE: cant have MO2 in FO4 install directory either as VFS has melt down and File path should not contain any spaces as the MKLink will have odd results

**Comment by Gernash** [comment]
and if you username has spaces (ZOMG) use ? instead of spaces

**Comment by WhiteZeroX** [comment]
But MO does work, just has this one odd issue as I've described. However, I'll try keeping everything for MO out of the FO4 dir and report back

**Comment by WhiteZeroX** [comment]
@Gernash yeah moving my mod install dir out of the Fallout 4 install dir didn't help, Fallout4.esm still gets added to plugins.txt. And I'm not going through the trouble of doing the symlinks, seeing as VFS does work fine for me, it just has this one oddity.

**Comment by Gernash** [comment]
Cool if you work out how to do it without a symbolic link then please post the resolution as I've been unable to find a way so far.

BTW where is the plugins.txt your talking about? in the pofiles or the game config folder c:\user\appdata\local\fallout4?

**Comment by Gernash** [comment]
my profile folder contains
# This file was automatically generated by Mod Organizer.
*Fallout4.esm
*DLCRobot.esm
*DLCworkshop01.esm
*DLCCoast.esm
*DLCworkshop02.esm
*DLCworkshop03.esm
*DLCNukaWorld.esm

**Comment by WhiteZeroX** [comment]
@gernash profiles folders
